core/src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -893,4 +893,33 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.endObject();
         return builder;
     }
+
+    @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        String newLine = System.getProperty("line.separator");
+
+        result.append(this.getClass().getName());
+        result.append(" Object {");
+        result.append(newLine);
+
+        Field[] fields = this.getClass().getDeclaredFields();
+
+        for (Field field: fields) {
+            result.append("  ");
+            try {
+                result.append(field.getName());
+                result.append(": ");
+                result.append(field.get(this));
+            }
+            catch (IllegalAccessException ex) {
+                System.out.println(ex);
```

Comment:
Writing to `System.out` isn't the right thing to do with an exception, and probably fails the build. The fix here is not to use reflection (see previous comment).